exp/addons/internal/controllers/clusterresourcesetbinding_controller.go

```diff
@@ -80,7 +80,12 @@ func (r *ClusterResourceSetBindingReconciler) Reconcile(ctx context.Context, req
 	if err != nil && !apierrors.IsNotFound(err) {
 		return ctrl.Result{}, err
 	}
-
+	// On a backup, the clusterResourceSetBinding will not have an ownerRef set
```

Comment:
nit 
No need to specify this is a check for the backup case - it's a check we should be doing in all cases.